### PR TITLE
Add missing TVL for TempleDAO using subgraph

### PR DESCRIPTION
**Pull request by medariox**
##### methodology (what is being counted as tvl, how is tvl being calculated):

Counts stables from TEMPLE-FRAX AMM LP, Temple Defend and FRAX+3CRV LP as treasury TVL

### CODE REVIEW

File: projects/templedao/index.js [hunk 1]
```diff
@@ -1,32 +1,39 @@
...
+  }
+}
+`
 
-  return balances;
+/*** TVL Portion (Treasury) ***
+ * Treasury TVL consists of stables from TEMPLE-FRAX AMM LP, Temple Defend and FRAX+3CRV LP
+ ***/
+async function ethTvl(timestamp, block) {
+  const queriedData = await request("https://api.thegraph.com/subgraphs/name/templedao/templedao-metrics", protocolQuery, {block})
```
Comment (g1nt0ki) [inline projects/templedao/index.js:23]: Not sure, why we need to switch to graph here. I see that old treasury contract does not have the LPs you mentioned, can you give me the (treasury) address, I could add them :)

Comment (medariox) [reply projects/templedao/index.js:23]: Hey, thanks for the quick feedback!
Using the subgraph is just a convenient way to combine multiple sources. That way we don't have to update the adapter every time we move the treasury funds.

If you still want to use the separate wallets instead, here are the addresses:
TEMPLE-FRAX AMM LP: `0x6021444f1706f15465bee85463bcc7d7cc17fc03`
Temple Defend:  `0x8a5058100e60e8f7c42305eb505b12785bba3bca + 0xb0d978c8be39c119922b99f483cd8c4092f0ea56`
FRAX+3CRV LP: `0x5c8898f8e0f9468d4a677887bc03ee2659321012`

Please let me know if you need anything else.

**Comment by llamatester** [comment]
The adapter at projects/templedao/index.js exports TVL: 
        
 
 
    ethereum                  146.48 M
    ethereum-staking          69.86 M
    staking                   69.86 M
    
    total                    146.48 M

**Comment by g1nt0ki** [comment]
I am closing this in favor of https://github.com/DefiLlama/DefiLlama-Adapters/pull/2296 which includes update to templedao adapter

**Pull request closed without merging**